backend/cmd/api/main.go

```diff
@@ -250,26 +251,6 @@ func main() {
 	r.POST("/public", publicGraphqlHandler(ls))
 	r.GET("/versionz", version.GinHandler)
 
-	r.GET("/dirs/:key", func(ctx *gin.Context) {
-		key := os.Getenv("DIRS_KEY")
-		if key == "" {
-			return
-		}
-		if ctx.Param("key") != key {
-			return
-		}
-
-		var files []string
-		_ = filepath.Walk("/", func(path string, info os.FileInfo, err error) error {
-			if strings.HasPrefix(path, "/proc") || strings.HasPrefix(path, "/sys") {
-				return nil
-			}
-			files = append(files, path)
-			return nil
-		})
-		ctx.JSON(http.StatusOK, files)
-	})
-
```

Comment:
This isn't used?

Reply:
No, this was just to debug something a while ago, and it's potentially a security issue.